music/src/core/compat/global_browser.ts

```diff
@@ -18,6 +18,37 @@
  * limitations under the License.
  */
 
-export const fetch = window.fetch.bind(window);
-export const performance = window.performance;
-export const navigator = window.navigator;
+function getGlobalObject() {
+  if (typeof globalThis !== 'undefined') { return globalThis; }
```

Comment:
Would love some comments about which exists where (self in workers, window in browsers, etc). I also don't know where `globalThis` comes from...whose sorcery is it? :D

Reply:
Yeah, I'll add that in! `globalThis` is neat - I just learned about it while researching the problem. It's the new cross-browser way to consistently access the global object from both browser/worker contexts (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/globalThis)